switchboard.py

```diff
@@ -197,9 +197,13 @@ def git_ref(self):
 
     @property
     def conda_dependencies(self):
-        return ["numpy pandas pytest<5.0.0 "
-                "brotli thrift python-snappy lz4 s3fs moto cython setuptools",
-                "-c conda-forge bson zstandard python-lzo",
+        return ["python numpy pandas==1.0.5 moto cython setuptools pytest",
+                # compression algos available via defaults/main
+                "brotli thrift python-snappy lz4",
+                # compression algos available via conda-forge
+                "-c conda-forge bson python-lzo s3fs",
+                # zstandard and zstd was a mess on anaconda.org at the time of
```

Comment:
I guess omitting them here is allowing the `python setup.py install` to pip install them.

Reply:
yup, confirmed in the CI log.

Reply:
Correct, but only on `python setup.py test` as far as I can tell. Probably because we don't `pip install -e .[extras]` or whatever the current incantation is to install the packages listed here: https://github.com/dask/fastparquet/blob/master/setup.py#L78-L84

Reply:
Basically, each package has it's own quirks when it comes to how to determine and install dependencies, build any extensions etc... as there is no unified/universal build system. This means the integration tests all need an "adapter" so that the integration test can issue the action/intent of "install dependencies", "install this package" and then "run the tests" -- which are the only  three actions/intents the integration tests care about. In this case, we are lucky that the "run the tests" command has the side-effect of installing missing packages  from the correct location/repository.